src/renderers/webgl/plugins/LensFlarePlugin.js

```diff
@@ -337,13 +339,15 @@ THREE.LensFlarePlugin = function ( renderer, flares ) {
 			screenPositionPixels.x = screenPosition.x * halfViewportWidth + halfViewportWidth;
 			screenPositionPixels.y = screenPosition.y * halfViewportHeight + halfViewportHeight;
 
+			areaToCopy.x = viewport.x + screenPositionPixels.x - 8; // horizontal coordinate of the lower left corner of the pixels to copy
+			areaToCopy.y = viewport.y + screenPositionPixels.y - 8; // vertical coordinate of the lower left corner of the pixels to copy
+
 			// screen cull
 
-			if ( hasVertexTexture || (
```

Comment:
I see I see.

Actually, we may want to remove the `MAX_VERTEX_TEXTURE_IMAGE_UNITS === 0`  code path. WebGLStats says they're only 1.8%.